src/ApiService/ApiService/OneFuzzTypes/ReturnTypes.cs

```diff
@@ -2,6 +2,16 @@
 
 namespace Microsoft.OneFuzz.Service {
 
+    public static class Result {
+        public readonly record struct OkPlaceholder();
+        public readonly record struct OkPlaceholder<T>(T Value);
+        public readonly record struct ErrPlaceholder<T>(T Value);
+
+        public static OkPlaceholder Ok() => new();
+        public static OkPlaceholder<T> Ok<T>(T value) => new(value);
+        public static ErrPlaceholder<T> Error<T>(T value) => new(value);
```

Comment:
This is an improvement-in-passing to improve the ergonomics of using the result types. See the changes in `Orm.cs` below.